packages/embark-vyper/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "embark-vyper",
+  "version": "4.0.0-beta.0",
+  "author": "Iuri Matias <iuri.matias@gmail.com>",
+  "contributors": [],
+  "description": "Embark is a framework that allows you to easily develop and deploy DApps",
+  "homepage": "https://embark.status.im/",
+  "keywords": [
+    "blockchain",
+    "dapps",
+    "ethereum",
+    "ipfs",
+    "serverless",
+    "solc",
+    "solidity"
+  ],
+  "license": "MIT",
+  "repository": "github:embark-framework/embark",
+  "main": "./dist/index.js",
+  "scripts": {
+    "build": "cross-env BABEL_ENV=node babel src --extensions \".js,.ts\" --out-dir dist --root-mode upward --source-maps",
+    "ci": "npm run qa",
+    "clean": "npx rimraf dist embark-*.tgz package",
+    "lint": "npm-run-all lint:*",
+    "lint:js": "eslint src/",
+    "lint:ts": "tslint -c tslint.json \"src/**/*.ts\"",
+    "package": "npm pack",
+    "qa": "npm-run-all lint typecheck build package",
+    "reset": "npm run clean && npx rimraf node_modules",
+    "start": "npm run watch",
+    "typecheck": "tsc",
+    "watch": "run-p watch:*",
+    "watch:build": "npm run build -- --verbose --watch",
+    "watch:typecheck": "npm run typecheck -- --preserveWatchOutput --watch"
+  },
+  "dependencies": {
+    "async": "2.6.1",
+    "@babel/preset-typescript": "7.1.0",
+    "shelljs": "0.5.3"
+  },
+  "devDependencies": {
+    "typescript": "3.1.3"
```

Comment:
needs the same devDeps as were added for `embark-compiler`, has to do w/ how things like `rimraf` get put on the PATH by `npm|yarn run` even though the yarn workspace hoists the modules into root `node_modules`.